src/Common/AzurePSCmdlet.cs

```diff
@@ -57,12 +58,6 @@ public abstract class AzurePSCmdlet : PSCmdlet, IDisposable
         protected IList<Regex> _matchers { get;  private set; }  = new List<Regex>();
         private static readonly Regex _defaultMatcher = new Regex("(\\s*\"access_token\"\\s*:\\s*)\"[^\"]+\"");
 
-        // Using Ansi Code to control font color(97(Bold White)) and background color(0;120;212(RGB))
-        private static readonly string ansiCodePrefix = "\u001b[97;48;2;0;120;212m";
-
-        // using '[k' for erase in line. '[0m' to ending ansi code
-        private static readonly string ansiCodeSuffix = "\u001b[K\u001b[0m";
```

Comment:
Aligned the style of intercept survey with highlighted information. Previous style is hardcoded to match the same Azure branding color. Since it does not get adjusted when a user customizes theme, it is not desired.